src/dex-ui/pages/utils.ts

```diff
@@ -0,0 +1,29 @@
+export const isValidUrl = (urlString: string) => {
```

Comment:
These functions are generically useful. We may want to move these functions to the top-level shared utils folder `@utils` if there used across multiple components.